app/build.gradle

```diff
@@ -33,7 +32,7 @@ android {
     defaultConfig {
         applicationId "car.pace.cofu"
         minSdkVersion 23
-        targetSdkVersion 30
+        targetSdkVersion 31
```

Comment:
@Melle010 does this mean, a user needs an Android 12 device to run the app?

Reply:
No, this basically is just an information for `gradle` that the app has been tested/targeting Android 12 so Android can do some backwards-compatibility magic, if needed.

We are also compiling the app with API 31 (line 21)

Two lines above, the `minSdkVersion` defines the lowest SDK where the app still runs on, which is still `23`.